jobs/payment-jobs/tasks/statement_notification_task.py

```diff
@@ -78,14 +80,27 @@ def send_notifications(cls):
             params['frequency'] = statement.frequency.lower()
             # logic changed https://github.com/bcgov/entity/issues/4809
             # params.update({'url': params['url'].replace('orgId', payment_account.auth_account_id)})
+
+            notification_success = True
+            eft_enabled = flags.is_on('enable-eft-payment-method')
```

Comment:
I'd recommend using the default `flags.is_on('enable-eft-payment-method', False)`

Reply:
makes sense, updated to specify the default explicitly

Reply:
thank you

![image](https://github.com/bcgov/sbc-pay/assets/3484109/47a4c071-32b1-4948-804c-282b0b3a4033)